src/modules/format-utils.js

```diff
@@ -204,7 +204,31 @@ function timeLeft(in_date) {
     return `in ${oxfordStringifyValues(labels, 'and')}`;
 }
 
+
+/**
+ * Unescapes HTML entities (&#\d+;) only
+ * @param {String} str String containing HTML numeric entities
+ * @returns {String} An unescaped string
+ */
+function unescapeEntities(str) {
+    if (typeof str === 'object') {
+        try {
+            str = str.valueOf();
+        }
+        catch ( error ) {
+            throw new TypeError('Utils: tried turning argument into a string, unsuccessful');
```

Comment:
Maybe prefix this message onto the existing error message and re-throw, to avoid muting the caught exception:

```js
error.message = 'Utils: ... :\n' + error.message;
throw error;
```